packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -165,11 +183,12 @@ export const remarkShakuCodeAnnotate = (
             const shouldHighlight =
               isHighlightingBlock || shouldHighlighNextSourceLine;
             shouldHighlighNextSourceLine = false;
-
+            const shouldDim = isDimBlock || shouldDimNextSourceLine;
+            shouldDimNextSourceLine = false;
             const sourceLine = line.line;
             const prefix = `<div class="line${
               shouldHighlight ? " highlight" : ""
-            }">`;
+            } ${shouldDim ? "dim" : ""}">`;
```

Comment:
this causes extra space for all lines if not dim

Reply:
Thanks! The extra spaces have been removed.